server/src/user/user.e2e-spec.ts

```diff
@@ -0,0 +1,44 @@
+import * as request from 'supertest';
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import { UserModule } from './user.module';
+import { UserService } from './user.service';
+import { AuthType } from '@prisma/client';
+
+describe('UserModule E2E', () => {
+  let app: INestApplication;
+  let moduleRef: TestingModule;
+
+  beforeAll(async () => {
+    moduleRef = await Test.createTestingModule({
+      imports: [UserModule],
+    }).compile();
+
+    app = moduleRef.createNestApplication();
+    await app.init();
+  });
+
+  it(`Create and check user`, async () => {
```

Comment:
Great infrastructure! This will make it easy for us to do test-driven development!